rdtools/test/analysis_chains_test.py

```diff
@@ -179,12 +179,25 @@ def test_sensor_analysis_ad_hoc_filter(sensor_parameters):
         rd_analysis.sensor_analysis(analyses=['yoy_degradation'])
 
 
+def test_sensor_analysis_daily_ad_hoc_filter(sensor_parameters):
```

Comment:
This looks good to me!  I think if we want complete test coverage we should also have daily analogs for:
- `test_filter_components`
- `test_filter_components_no_filters`
- `test_filter_ad_hoc_warnings`

Reply:
Added associated unit tests for the daily filter by https://github.com/NREL/rdtools/pull/348/commits/fde7cae818b3c5bbe015e5659a35b44d7df67de9